src/Components/SampleModelsControl.jsx

```diff
@@ -152,13 +152,13 @@ const useStyles = makeStyles({
       borderColor: 'gray',
     },
     '& .MuiOutlinedInput-root.Mui-focused .MuiOutlinedInput-input': {
-      color: 'green',
-    },
-    '& .MuiInputLabel-root.Mui-focused': {
-      color: 'green',
+      color: 'blue',
     },
+    // '& .MuiInputLabel-root.Mui-focused': {
```

Comment:
same

Reply:
For both, please convert to:
```
TODO(oleg): ... comment ...
```
so we can find later